tests/test_indices.py

```diff
@@ -751,6 +751,54 @@ def test_standardized_precipitation_evapotranspiration_index(
 
         np.testing.assert_allclose(spei.values, values, rtol=0, atol=diff_tol)
 
+    # reference results: Obtained with R package `standaRdized`
```

Comment:
@SarahG-579462 @coxipi
I could'nt find any other reference to test against for SSI (same for SGI I think). I could only replicate two conditions out of many. When the values get too far from zero, `standaRdized` outputs lots of NaNs and the non-NaNs are far away from us. Also in these replicable values, the offset is somewhat constant.

It looks like the problem is on their side?

Do you think this is still a valuable test then?

I'll also add a couple tests against ourselves.